README.md

```diff
@@ -1,28 +1,92 @@
-# Jina Documentations
+# Jina Documentation
 
 ![CD](https://github.com/jina-ai/docs/workflows/CD/badge.svg?branch=docs-migration)
 
-Please read the documentations at https://docs.jina.ai 
 
-This repository hosts the documentation of Jina. It gets updated when [jina-ai/jina](https://github.com/jina-ai/jina/) is modified.
+> “Open source is this magical thing right? You release code, and the code gnomes come and make it better for you.
+>
+> Not quite.
+> There are lots of ways that open source is amazing, but it doesn’t exist outside the laws of physics. You have to put work in, to get work out.
+>
+>You only get contributions after you have put in a lot of work.
+>You only get contributions after you have users.
+>You only get contributions after you have documentation.”
+>
+From @ericholscher [guide to writing docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
 
-All doc-related issues should be [submitted to here](https://github.com/jina-ai/jina/issues/new).
+## Table of contents
+*  [Hierarchical structure](#Hierarchical)
+* [Missing content](#Missing)
+*  [How to add pages](#howto)
+*  [Jina AI style guide](#style)
+* [Updating Docstrings](#docstrings)
+* [Updating API](#apis)
+* [Technical aspects of this site](#tech)
 
-## Read the Docs Locally
+## Hierarchical structure <a name="Hierarchical"></a>
 
-You can read the documentations locally via:
+Jina docs adheres to the following hierarchical structure. Each Jina product has its own section, containing three subsections below.
```

Comment:
The table seems not well displayed?
![image](https://user-images.githubusercontent.com/44033547/107807287-e786d680-6d35-11eb-9ca7-9296ccad1fd1.png)

Reply:
s/adheres/adhere

or:

s/docs/documentation

(just in case `s/foo/bar` is a bit obscure, it means **s**ubstitute `foo` for `bar` (I was just in vim and fingers are on autopilot)